### ISSUE TITLE
UI: Dictionary feedback

### ISSUE THREAD

**Issue by kcullion** [nci-hcmi-catalog/portal#609]
- [x] Add space underneath tab container, above footer
- [ ] if admin makes a few changes then reverts all changes, put dictionary in the state it was in before any changes/edits (like the reset state)
<img width="1439" alt="hcmi-dictionary-feedback1" src="https://user-images.githubusercontent.com/18424138/82371599-7570a380-99e8-11ea-97ad-e8d4032a1070.png">

- [x] For the publish all updates modal, please change the body copy to. 
"These updates will be applied to all applicable models in any state. Any published models that have been affected will switch to "Changes not yet published". You can republish them in the model management section. "

- [x] Not sure if this is part of this ticket....but we alphabetized the values in the dictionary. Can we do the same in "add/edit" model form? (for all fields with value lists)
<img width="573" alt="Screen Shot 2020-05-20 at 10 26 52 AM" src="https://user-images.githubusercontent.com/18424138/82458313-87078900-9a84-11ea-9173-d7a38ccc3019.png">

- [x] display dates side by side like "Last published: date | Last updated:date"

- [x] The affected model count isn't working on the confirmation message. Note: if I just add new values or make changes that don't affect any models, "undefined" would say "zero"
<img width="1476" alt="Screen Shot 2020-05-20 at 10 44 32 AM" src="https://user-images.githubusercontent.com/18424138/82460316-039b6700-9a87-11ea-8001-6cebbecb1327.png">

NICE TO HAVE (only if time)
- [x] add the orange circle indicator for all values that have dependent value updates. And the circle is removed if the editor reverts the changes back to none
<img width="491" alt="Screen Shot 2020-05-20 at 10 32 44 AM" src="https://user-images.githubusercontent.com/18424138/82460131-c9ca6080-9a86-11ea-9b73-1306c8a47449.png">

**Comment by mistryrn** [comment]
@kcullion the second point will require a bigger change to the API, since as soon as you start making any changes it creates a draft. It doesn't "reset" the draft automatically when you undo all your changes. Gonna loop @joneubank in on this to see his thoughts.

**Comment by kcullion** [comment]
ok thanks @mistryrn ... looping @rosibaj in to see if it's worth fixing

**Comment by kcullion** [comment]
Thanks @mistryrn !
Some fixes:

- [ ] When the value name is long, the "expand all" button changes width and wraps on two lines. Can you just make it so the value title spans the two lines
<img width="576" alt="Screen Shot 2020-05-20 at 12 28 32 PM" src="https://user-images.githubusercontent.com/18424138/82472045-904d2180-9a95-11ea-967e-dc4b47afaf86.png">

**Comment by rosibaj** [comment]
Regarding the outstanding topic above:
- this is an edge case
- That will be tricky to handle, basically the click/mouse down event (how we know to select the other parent) is triggered before the “blur” event (how we know to save the dependent field you’re editing), so the dependent field doesn’t get saved before you click the other parent value.
- If we change the selection to occur on an event that fires after blur, we’ll bring up some bugs related to clicking the save button on an editable field since the blur will happen, it will save, and then the click will happen and it’ll go back into edit mode.
- basically, the user is moving focus state to focus state without allowing a save, when there are multiple save mechanisms (click anywhere else, or click the save icon). 
- we will not do anything as there are too many other issues with changing the order of operations

**Comment by rosibaj** [comment]
thanks for edits @mistryrn! looks great.
